ci-operator/config/kubev2v/migration-planner-ui-app/kubev2v-migration-planner-ui-app-master.yaml

```diff
@@ -0,0 +1,30 @@
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.24.0
+promotion:
+  to:
+    - name: "4.20"
+      namespace: ocp
+releases:
+  initial:
+    integration:
+      name: "4.20"
+      namespace: ocp
+  latest:
+    integration:
+      include_built_images: true
+      name: "4.20"
+      namespace: ocp
```

Comment:
Why is this needed?
Does it cause to the auto release branch creation?
Can this be removed?